src/inc/winwrap.h

```diff
@@ -981,7 +981,11 @@ inline int LateboundMessageBoxW(HWND hWnd,
 #if defined(FEATURE_CORESYSTEM) && !defined(CROSSGEN_COMPILE)
     // Some CoreSystem OSs will support MessageBoxW via an extension library. The following technique is what
     // was recommeded by Philippe Joubert from the CoreSystem team.
-    HMODULE hGuiExtModule = WszLoadLibrary(W("ext-ms-win-ntuser-gui-l1"), NULL, 0);
+    HMODULE hGuiExtModule = WszLoadLibrary(W("ext-ms-win-ntuser-gui-l1-1-0"), NULL, 0);
```

Comment:
I think we should get rid of the internal Windows API set loading. Just load the user32 (basically remove the ifdef and update the comment).